docs/platform/self_hosting/configuration.mdx

```diff
@@ -129,61 +233,212 @@ tolgee.smtp.username=*****************
 tolgee.smtp.password=*****************
 tolgee.smtp.port=465
 tolgee.smtp.auth=true
-tolgee.smtp.sslEnabled=true
+tolgee.smtp.ssl-enabled=true
 tolgee.smtp.from=Tolgee <no-reply@tolgee.yourserver.something>
 ```
 
 ### host
-Smtp host
+SMTP host
 ### username
-The username for smtp authentication
+The username for SMTP authentication
 ### password
-Password for smtp authentication
+Password for SMTP authentication
 ### port
+SMTP port
 
 ### auth
 Whether authentication is enabled. (default: `false`)
-### tlsEnabled
-Whether tls is enabled (default: `false`)
-### sslEnabled
-Whether ssl is enabled (default: `false`)
-### tlsRequired
-Whether ssl is required (default: `false`)
+### tls-enabled
+Whether TLS is enabled .(default: `false`)
+### ssl-enabled
+Whether SSL is enabled. (default: `false`)
+### tls-required
+Whether SSL is required. (default: `false`)
 ### from
 The sender name and address in standard SMTP format.
 
-## LDAP authentication properties
-This feature is currently experimental! (prefix: tolgee.authentication.ldap)
- - enabled
- - port
- - urls
- - baseDn
- - securityPrincipal
- - principalPassword
- - userDnPattern
+## Machine translation
```

Comment:
We currently don't have frontend prepared for these features. Can you just remove this for now and I will add it when we merge https://github.com/tolgee/server/pull/920